libp2p/RLPXFrameCoder.h

```diff
@@ -1,59 +1,38 @@
-/*
- This file is part of cpp-ethereum.
- 
- cpp-ethereum is free software: you can redistribute it and/or modify
- it under the terms of the GNU General Public License as published by
- the Free Software Foundation, either version 3 of the License, or
- (at your option) any later version.
- 
- cpp-ethereum is distributed in the hope that it will be useful,
- but WITHOUT ANY WARRANTY; without even the implied warranty of
- MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- GNU General Public License for more details.
- 
- You should have received a copy of the GNU General Public License
- along with cpp-ethereum.  If not, see <http://www.gnu.org/licenses/>.
- */
-/** @file RLPXFrameCoder.h
- * @author Alex Leverington <nessence@gmail.com>
- * @date 2015
- */
-
+// Aleth: Ethereum C++ client, tools and libraries.
+// Copyright 2019 cpp-ethereum Authors.
+// Licensed under the GNU General Public License v3.
 
 #pragma once
 
-#include <memory>
-#include <libdevcore/Guards.h>
-#include <libdevcrypto/CryptoPP.h>
 #include "Common.h"
+#include <libdevcore/RLP.h>
+#include <libdevcrypto/CryptoPP.h>
+#include <memory>
 
 namespace dev
 {
 namespace p2p
 {
 
-struct RLPXFrameDecryptFailed: virtual dev::Exception {};
-
 /**
  * @brief Encapsulation of Frame
  * @todo coder integration; padding derived from coder
  */
 struct RLPXFrameInfo
 {
-	RLPXFrameInfo() = default;
```

Comment:
The build fix is actually only here. You cannot create the default constructor while having `const` fields.